crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -117,6 +116,22 @@ impl Analyzer<'_, '_> {
             | Type::EnumVariant(..)
             | Type::Param(_)
             | Type::Module(_) => return Ok(ty),
+            Type::Interface(Interface {
```

Comment:
I think this approach is wrong

Reply:
Did you mean we should add more condition here to decrease the range of match arm?
Or It is totally wrong location to implement?

Reply:
Can you give any tips like where to start first?

Reply:
I think some match expressions should be added to https://github.com/dudykr/stc/blob/4033e8b3a1803640fdcf409442a3cc91b900f237/crates/stc_ts_file_analyzer/src/analyzer/assign/mod.rs#L1036